_posts/2019-04-24-accessing-kafka-part-2.md

```diff
@@ -0,0 +1,317 @@
+---
+layout: post
+title:  "Accessing Kafka: Part 2 - Node ports"
+date: 2019-04-17
+author: jakub_scholz
+---
+
+In the second part of this blog post series we will look at exposing Kafka using node ports.
+This post will explain how node ports work and how can they be user with Kafka.
+It will also cover the different configuration options which are available to the users and cover the main pros and cons of using node ports.
```

Comment:
```suggestion
It will also cover the different configuration options which are available to users and cover the main pros and cons of using node ports.
```